core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -585,6 +585,21 @@ package object config {
       .intConf
       .createWithDefault(128)
 
+  private[spark] val LISTENER_BUS_LOG_SLOW_EVENT_ENABLED =
+    ConfigBuilder("spark.scheduler.listenerbus.logSlowEvent.enabled")
```

Comment:
Since this is more like debug or dev feature, `.internal()`?

Reply:
Just curious: if we set the configuration to `.internal()`, who's the target seeing this config and changing it? If the value is true by default then it may not matter much whether this is marked as internal, but if the value is false, I'm not sure how end users can find the config and enable it.

Reply:
I would be expecting the config managed by the system admin that maintains the cluster. Since this is config in Core, the end users are not able to change it dynamically. Maybe I would add the point that this config is only targeted for expert users of Spark, so mark it `internal()` sounds reasonable to me.